### PR TITLE
Extend Hiera 5 template to support trocla backend

### PR DESCRIPTION
**Pull request by m4rx0**
#### Pull Request (PR) description
To be able to configure the custom Hiera backend trocla [1] I had to extend the Hiera 5 `hiera.yaml` template. May there's a better way to do that. So, please let me know.

[1] https://github.com/duritong/puppet-trocla#hiera-backend

### CODE REVIEW

File: templates/hiera.yaml.epp [hunk 1]
```diff
@@ -61,7 +61,15 @@ hierarchy:
 <% if $hr['options'] { -%>
     options:
     <%- $hr['options'].each |$key,$value| { -%>
+    <%- if $value =~ Array { -%>
```
Comment (rnelson0) [inline templates/hiera.yaml.epp]: This block (through L72, I think) should have an extra indent.

File: templates/hiera.yaml.epp [hunk 2]
```diff
@@ -61,7 +61,15 @@ hierarchy:
...
+      <%= $key %>:
+        <%- $value.each |$value| { -%>
+        - "<%= $value %>"
+        <%- } -%>
+    <%- } -%>
+    <%- unless $value =~ Array { -%>
```
Comment (rnelson0) [inline templates/hiera.yaml.epp]: I think this should be part of an `else` clause, as you want one or the other.

File: templates/hiera.yaml.epp (hunk 1 above)
Comment (m4rx0) [reply templates/hiera.yaml.epp]: ok, I made an extra indent.

File: templates/hiera.yaml.epp (hunk 2 above)
Comment (m4rx0) [reply templates/hiera.yaml.epp]: ok, done

**Comment by m4rx0** [comment]
Hi @rnelson0 ,
can you please review again.

**Comment by rnelson0** [comment]
Thank you!

**Pull request merged**